### PR TITLE
Added ability to list records with colors

### PR DESCRIPTION
**Pull request by flakerimi**
Might not be elegant solution but it does the job. Open for better solution.

screen shot 2018-01-04 at 16 17 04

### CODE REVIEW

**Comment by LukeTowers** [comment]
A better solution exists, it's called `type: partial` :) This would have to be more complex because you would need to evaluate whether to use a dark or a light text colour for the overlaid text based on the selected colour.

**Comment by flakerimi** [comment]
I saw dark/light problem, but on hover you can see both, maybe just a <span style="width: 50px; height: 50px; background: $value></span> $value, so you have that before text. 

I believe its better to have type: colorpicker rather than partial in this case since it it already as type.

**Comment by LukeTowers** [comment]
escape the value with `e($value)` and I'd accept it

File: modules/backend/widgets/Lists.php [hunk 1]
```diff
@@ -1164,7 +1164,17 @@ protected function evalTimetenseTypeValue($record, $column, $value)
 
         return Backend::dateTime($dateTime, $options);
     }
+    /**
+     * Process as background color, to be seen at list
+     */
+    protected function evalColorPickerTypeValue($record, $column, $value)
+    {
+        if (is_array($value) && count($value) == count($value, COUNT_RECURSIVE)) {
+            $value = implode(', ', $value);
```
Comment (LukeTowers) [inline modules/backend/widgets/Lists.php]: What is the purpose of this line?

**Comment by flakerimi** [comment]
Done.

**Comment by LukeTowers** [comment]
Thanks @flakerimi! Could you add this to the documentation? https://github.com/octobercms/docs

**Comment by flakerimi** [comment]
Its already there, https://github.com/octobercms/docs/blob/master/backend-forms.md#color-picker
No need for further explanation since is assumed behaviour. If you use it in fields, you can use it in columns too.

**Comment by LukeTowers** [comment]
That's not true @flakerimi: http://octobercms.com/docs/backend/lists#column-types

**Pull request merged**